File: java/IdenticalTwins.java
// link-https://workat.tech/problem-solving/practice/identical-twins

class Solution {
	int getIdenticalTwinsCount (int[] arr) {
        // add your logic here
        int ans=0;

        for(int i=0;i<arr.length;i++){
            for(int j=i+1;j<arr.length;j++){
                if(arr[i]==arr[j]) ans+=1;
            }
        }
        return ans;

    }
}

File: python/vedant_bhatnagar/Work@Tech/Basic Questions/IdenticalTwins.py
class Solution:
	def getIdenticalTwinsCount(self, arr: List[int]) -> int:
		# add your logic here
		count = 0
		n = len(arr)
		for i in range(n):
			for j in range(i+1, n):
				if arr[i] == arr[j]:
					count = count+1
		return count



